src/contracts/pods/EigenPod.sol

```diff
@@ -271,41 +228,63 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
      */
     function verifyBalanceUpdate(
         uint40 validatorIndex,
-        BeaconChainProofs.ValidatorFieldsAndBalanceProofs calldata proofs,
+        BeaconChainProofs.BalanceUpdateProofs calldata proofs,
         bytes32[] calldata validatorFields,
         uint256 beaconChainETHStrategyIndex,
-        uint64 oracleBlockNumber
+        uint64 oracleTimestamp
     ) external onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_BALANCE_UPDATE) {
-       // ensure that the blockNumber being proven against is not "too stale", i.e. that the validator's balance *recently* changed.
-        require(oracleBlockNumber + VERIFY_BALANCE_UPDATE_WINDOW_BLOCKS >= block.number,
-            "EigenPod.verifyBalanceUpdate: specified blockNumber is too far in past");
+       // ensure that the timestamp being proven against is not "too stale", i.e. that the validator's balance *recently* changed.
```

Comment:
`validatorFields` is not verified which seems broken. in fact we should just need to verify the `pubkeyHash`